pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -394,12 +394,21 @@ type ETCDEncryptionKeyRotation struct {
 	LastInitiationTime *metav1.Time `json:"lastInitiationTime,omitempty" protobuf:"bytes,3,opt,name=lastInitiationTime"`
 	// LastInitiationFinishedTime is the recent time when the ETCD encryption key credential rotation initiation was
 	// completed.
+	// Deprecated: This field will be removed in a future release. The field will be no longer needed with
```

Comment:
Can we add an _empty line_ between the _docstring_ and the _deprecation_ note ?

```suggestion
	//
	// Deprecated: This field will be removed in a future release. The field will be no longer needed with
```